### DISCUSSION TITLE
wrong installation instructions?

### DISCUSSION THREAD

**Discussion by VictorG238** [celisej567/Fake-CSM-Source-2013#4] [Q&A] [answered]
i dont what to sound rude but how the hell is this supposed to be install, you put the files and all that then the instructions say to open the files "client_base.vpc and server_base.vpc" but these are nowhere to be found, at first I thought it would be in the SDK 2013 BIN folder but they are not, then it says I am supposed to modify these files to add "$Include "csm. vpc", but the aforementioned files are not there and even though I add the env_cascade_light in hammer with the FGD and do all the RGB configuration etc etc it still doesn't work and how am I supposed to create a . bat file called "createallprojects.bat" and add nothing else -f to the end and that compiles the files if there is no command list for it or anything and it doesn't come in the main build or mapbase 7.1 build files how is it supposed to work. am i missing something? because i create my mods with the XBLAH modding tool and from what i researched that .bat is for compiling the mods files

**Comment by celisej567** [comment, marked as answer]
You need this [https://github.com/mapbase-source/source-sdk-2013](https://github.com/mapbase-source/source-sdk-2013). In src/game/client or server/here_is_base.vpc
